pkg/minikube/machine/start.go

```diff
@@ -155,6 +155,8 @@ func createHost(api libmachine.API, cfg *config.ClusterConfig, n *config.Node) (
 	h.HostOptions.AuthOptions.StorePath = localpath.MiniPath()
 	h.HostOptions.EngineOptions = engineOptions(*cfg)
 
+	api.DefineGuest(h)
```

Comment:
what does line 158 do?

Reply:
This is where we set the host's guest operating system. The complains were as a result of some failing tests